.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: pnpm/action-setup@v2
         name: Install pnpm
         with:
-          version: 8.10.3
+          version: 8.15.1
```

Comment:
@vstefanovic97 is this necessary for this PR? Can we do this bump separately?

Reply:
@SergeAstapov it's necessary, see https://github.com/ember-cli-code-coverage/ember-cli-code-coverage/pull/405#issuecomment-1916394358